apps/schools/migrations/0023_auto_20170630_0642.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
```

Comment:
It can't be run as a migration because `ALTER TYPE` statements are NOT allowed to be run within a transaction. While newer Django version support disabling of transaction per migration file, 1.7 doesn't have this feature. Hence, instead of hacking that feature in or spending time upgrading the Django version for dubdubdub, much easier to just run each statement one by one.